server/rematch/celery.py

```diff
@@ -6,6 +6,8 @@
 
 # set the default Django settings module for the 'celery' program.
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'rematch.settings')
+import django
+django.setup()
```

Comment:
It works with your cmd, but has to be documented or set somehow (not manually :)